python-checks/src/test/resources/checks/fieldDuplicatesClassName.py

```diff
@@ -14,6 +14,12 @@ class NestedClass:
     def fun(self):
       self.nestedClass = 5 # Noncompliant [[secondary=-2]] {{Rename field "nestedClass"}}
 #          ^^^^^^^^^^^
+      myClass = 1
```

Comment:
@ivandalbosco but there is no FP here as it's local variable, not field. You should prepend this code with `self.` to have FP (and btw this case is already tested on line 25).